src/core/or/policies.c

```diff
@@ -2780,7 +2780,8 @@ parse_short_policy(const char *summary)
         high = low;
         break;
       case '-':
-        high = (unsigned) tor_parse_ulong(next+1, 10, low, 65535, &ok, &next);
+        high = (unsigned)
+          tor_parse_ulong(next + 1, 10, low, 65535, &ok, &next);
```

Comment:
This looks odd to me. Unless we are low on horizontal space, then I see no need to do something like this. Getting rid of the space between `(unsigned)` and the function is fine IMO.